src/Lookup/InMemoryEntityLookup.php

```diff
@@ -18,6 +21,7 @@
 class InMemoryEntityLookup implements EntityLookup {
 
 	private $entities = array();
+	private $exceptions = array();
```

Comment:
The types are documented in the public places, and can be inferred here by any non tard tool.